spec/system/teacher_interface/check_answers_spec.rb

```diff
@@ -67,10 +67,74 @@
     end
   end
 
+  it "display banner" do
```

Comment:
Would be good to explain which banner we're testing here, but the test itself looks good!